larray/core.py

```diff
@@ -10130,12 +10130,22 @@ def read_sas(filepath, nb_index=None, index_col=None, na=np.nan, sort_rows=False
     return df_aslarray(df, sort_rows=sort_rows, sort_columns=sort_columns, fill_value=na)
 
 
+def _check_axes_argument(func):
+    def wrapper(*args, **kwargs):
+        if len(args) > 1 and isinstance(args[1], (int, Axis)):
+            raise ValueError("If you want to pass several axes or dimension lengths to {}, "
+                             "you must pass them as a list (using []) or tuple (using()).".format(func.__name__))
+        return func(*args, **kwargs)
```

Comment:
when wrapping a function, you need to copy its __name__ and __doc__ and possibly other stuff, or preferably, use functools.wraps on the wrapper.